error.go

```diff
@@ -22,8 +22,9 @@ type Error struct {
 	depth int
 }
 
+// Error returns the error in the format "code: message".
 func (e *Error) Error() string {
-	return e.Code.String() + ": " + strings.Join(e.Msg, ": ")
+	return e.Code.String() + ": " + e.Msg
 }
```

Comment:
@escalopa `e.Msg` is a list of messages and not just a single message unlike the previous versions.

Reply:
After this resolution, I will merge